docs/_source/_common/snippets/start_page.md

```diff
@@ -1,36 +1,85 @@
+
+2. Get your `API_URL`:
+
+   * If you are using Docker, it is the URL shown in your browser (by default `http://localhost:6900`)
+   * If you are using HF Spaces, it should be constructed as follows: `https://[your-owner-name]-[your_space_name].hf.space`
+
+3. Get your `API_KEY` by clicking on your user icon in the top right corner of the Argilla UI, selecting "My settings" and copying the API key.
+
+## 2. Install the SDK with pip
+
+```sh
+pip install argilla
+```
+
+## 3. Connect to your Argilla server
+
+Connect the SDK with the server using `rg.init`. Make sure to replace "<your-api-url>" and "<your-api-key>" with the values you obtained in the first step.
 
 ```python
-# install datasets library with pip install datasets
 import argilla as rg
-from datasets import load_dataset
 
-# load an Argilla Feedback Dataset from the Hugging Face Hub
-# look for other datasets at https://huggingface.co/datasets?other=argilla
-dataset = rg.FeedbackDataset.from_huggingface("argilla/oasst_response_quality", split="train")
+rg.init(
+    api_url="<your-api-url>",
+    api_key="<your-api-key>"
+)
+```
+
+## 4. Create your first dataset
+
+1. Specify a workspace where the dataset is to be saved. You can check the available workspaces with the following snippet:
 
-# push the dataset to Argilla
-dataset.push_to_argilla("oasst_response_quality")
+```python
+workspaces = rg.Workspace.list()
+workspace_names = [workspace.name for workspace in workspaces]
+workspace_names
 ```
-:::
 
-:::{tab-item} Other datasets
+2. Create a FeedbackDataset with two labels ("sadness" and "joy"). Don't forget to replace "<your-workspace>" with the workspace you want to use:
 
 ```python
-# install datasets library with pip install datasets
-import argilla as rg
-from datasets import load_dataset
+dataset = rg.FeedbackDataset.for_text_classification(
+    labels=['sadness', 'joy'],
+    multi_label=False,
+    use_markdown=True,
+    guidelines=None,
+    metadata_properties=None,
+    vectors_settings=None,
+)
+dataset.push_to_argilla(name="my-first-dataset", workspace="<your-workspace>")
+```
 
-# load dataset from the hub
-dataset = load_dataset("argilla/gutenberg_spacy-ner", split="train")
+## 5. Add records
 
-# read in dataset, assuming its a dataset for token classification
-dataset_rg = rg.read_datasets(dataset, task="TokenClassification")
+Create a list with the records you want to add. Ensure that you match the fields with the ones specified in the previous step.
 
-# log the dataset
-rg.log(dataset_rg, "gutenberg_spacy-ner")
+```python
+records = [
+    rg.FeedbackRecord(
+        fields={
+            "text": "I am so happy today",
+        },
+    ),
+    rg.FeedbackRecord(
+        fields={
+            "text": "I feel sad today",
+        },
+    )
+]
+dataset.add_records(records)
 ```
-:::
-::::
\ No newline at end of file
+You can also use `pandas` or `load_dataset` to read an existing dataset and create records like above.
```

Comment:
Add a link to the most useful section to understand how to add records to a dataset